lib/elasticsearch_relation.rb

```diff
@@ -1,12 +1,11 @@
 # This class tries to reproduce an ActiveRecord relation
 class ElasticsearchRelation
   def initialize(params = {})
-    @params = params
+    @params = params.merge({ size: 10000 })
```

Comment:
I suggest the opposite: `{ size: 10000 }.merge(params)`, this way we can change it when calling